src/platform/site-wide/announcements/containers/Announcement.jsx

```diff
@@ -50,6 +50,18 @@ export class Announcement extends Component {
       .forEach(relatedAnnouncementName => {
         this.props.dismissAnnouncement(relatedAnnouncementName);
       });
+
+    // Manages focus on model close for screen reader accessibility
```

Comment:
I feel like there might be a better home for this to make it more general purpose, but I can't think of one right now.

Reply:
Beyond making this possibly more general purpose, I think there's another issue, and that is the tight coupling. I don't love how this Announcement component is aware of a specific announcement. It shouldn't need to be, ideally. I'd love to pass this functionality in as a callback (inversion of control).

Reply:
Also typo in comment ("model").

Reply:
updated code so that returnToFocus is passed in from the announcement config file.